packages/vue-vuetify/src/controls/IntegerControlRenderer.vue

```diff
@@ -36,7 +36,7 @@ import {
   type RendererProps,
 } from '@jsonforms/vue';
 import { defineComponent } from 'vue';
-import { VNumberInput } from 'vuetify/labs/VNumberInput';
+import { VNumberInput } from 'vuetify/components';
```

Comment:
Something must have changed. The `#numbers` example of the Vue Vuetify demo app no longer works properly

![Image](https://github.com/user-attachments/assets/d21ab1d4-ee6d-4fbc-bd03-0201233c34e3)